chrome/nav.js

```diff
@@ -1141,6 +1152,56 @@ function updateRoutePlanner( data ) {
 	}
 }
 
+function highlightVisited( data ) {
+	let ukey = Universe.getServer ( doc ).substr( 0, 1 );
+
+    // initial setup
+    if ( !data[ ukey + 'visit' ] ) {
+        data[ ukey + 'visit' ] = {};
+    }
+    if( !config.displayVisitedDecay ) {
+        decayTime = 180000; //3 minutes
+    } else {
+        var decayTime = 1000*config.displayVisitedDecay;
+    }
+    // directly save that we are on this tile;
+    data[ ukey + 'visit' ][ userloc ] = Date.now();
+    chrome.storage.local.set( data );
+
+    // highlight the tiles according to the time visited
+    var locs = Object.keys( data[ ukey + 'visit' ] );
+    var navtable = doc.getElementById( 'navareatransition' );
+	if ( !navtable )
+		navtable = doc.getElementById( 'navarea' );
+	if ( !navtable )
+		return;    
+    
+    var a = navtable.getElementsByTagName( 'a' );
+    var div = doc.createElement( 'div' );
+    for ( var i=0; i< a.length; i++ ) {
+        if (!a[i].getAttribute('onclick'))
+            continue;
+        let loc = a[i].getAttribute('onclick').split(/\(|\)/g)[1];
+        if ( locs.includes( loc ) ) {
+            let red = 0;
+            let green = 255;
+            let fade = Math.round((255 / decayTime) * ( Date.now() 
+                - data[ ukey + 'visit' ][ loc ] ) );
+            red += fade;
+            green -= fade;
+            setClass( a[i].parentNode );
+            setClass( a[i].firstChild );
+            
```

Comment:
could also do it as a function of opacity, something like 1 - ( 0.8 * ( time delta / decay threshold )  ) and a minimum, so that old tiles aren't as bright as recent ones. 

... fast decay near the recent times would probably be best to show greater differences in recently travelled tiles, but that's not really necessary